### PR TITLE
Issue 36 appveyor ci

### LINKED ISSUE #36: Configure AppVeyor CI build
Verify that our app builds on windows machines as well as on linux (travis)

### PR DESCRIPTION
**Pull request by aeldar**
Fixes #36

### CODE REVIEW

**Comment by aeldar** [comment]
If you gonna merge it, choose squash, to remove all trash, please :)

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+test_script:
+  - node --version
+  - npm --version
+  - npm run test:all
+
+after_test:
+  - npm run build
+
+before_deploy:
```
Comment (trioletas) [inline .appveyor.yml]: Is this required?

Comment (aeldar) [inline .appveyor.yml]: no, just a mistake. Removed.

**Comment by trioletas** [comment]
⭐️ thanks!

**Pull request merged**